python/lsst/daf/persistence/butlerFactory.py

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
I can do that but it'll touch maybe a dozen packages, and I think it would be more manageable to do it as part of a separate ticket.